url/resources/toascii.json

```diff
@@ -319,7 +319,7 @@
   },
   {
     "input": "\u04C0.com",
-    "output": null
+    "output": "xn--s5a.com"
   },
   {
     "comment": "This is U+2F868 (which is mapped to U+36FC starting with Unicode 16.0)",
```

Comment:
Could you update this comment while here to say "starting with Unicode 15.1.0" or maybe remove it altogether?

Reply:
OK, I think I'll just leave "This is U+2F868". What do you think about adding such comments to all surrogate pairs (for consistency)?

Reply:
It seems reasonable, although if you search for "\uD87E\uDC68" you also get this so not sure it's worth it.